### PR TITLE
feat: rename left and right to start and end

### LINKED ISSUE #3283: Rename `left` and `right` in the name or value of `props` to `start` and `end`
### Description

- If left or right is used in the name or value of props, change it to start or end.
	- props to be changed are marked `@deprecated` in v1.

### Problem Statement/Justification

v2.0 enhancement

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

Yes

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by nathanlao**
Closes #3283

## Description
Change `left` and `right` to `start` and `end` in the props

props to be changed are marked `@deprecated` in v1.

Docs changes in #3346

## Current behavior (updates)

value of props were using `left*` and `right*`

## New behavior

value of props are now using `start*` and `end*`

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by koralle** [comment]
The file search and viewing of the documentation showed that it had been properly corrected.

**Review by koralle — APPROVED** [review]

**Review by hirotomoyamada — CHANGES_REQUESTED** [review]
Documentation should be created in a separate PR.This will reflect in the documentation when it is imported.

**Comment by nathanlao** [comment]
@hirotomoyamada 

docs changeds had been done in #3346

File: packages/components/file-button/DOCS.json
Comment (hirotomoyamada) [inline packages/components/file-button/DOCS.json:1]: This file is an automatically generated file. Please revert this file.

File: packages/components/button/DOCS.json
Comment (hirotomoyamada) [inline packages/components/button/DOCS.json:1]: This file is an automatically generated file. Please revert this file.

File: packages/components/tag/DOCS.json
Comment (hirotomoyamada) [inline packages/components/tag/DOCS.json:1]: This file is an automatically generated file. Please revert this file.

**Comment by nathanlao** [comment]
@hirotomoyamada 

Auto-generated files have been removed

File: packages/components/button/src/button.tsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable perfectionist/sort-objects */
```
Comment (hirotomoyamada) [inline packages/components/button/src/button.tsx:1]: Remove this line.

File: packages/components/button/src/button.tsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+/* eslint-disable perfectionist/sort-interfaces */
```
Comment (hirotomoyamada) [inline packages/components/button/src/button.tsx:2]: Remove this line.

File: packages/components/tag/src/tag.tsx [hunk 3]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable perfectionist/sort-objects */
+/* eslint-disable perfectionist/sort-interfaces */
```
Comment (hirotomoyamada) [inline packages/components/tag/src/tag.tsx:2]: Remove this line.

**Comment by nathanlao** [comment]
@hirotomoyamada I have removed them, please re-review.

**Review by hirotomoyamada — DISMISSED** [review]

**Pull request merged**